### PR TITLE
Coinex transfer

### PR DESCRIPTION
**Pull request by Dan-krm**
Added transfer to Coinex:
```
coinex.transfer (USDT, 10, swap, spot)
2022-05-04T03:03:42.394Z iteration 0 passed in 334 ms

{
  id: undefined,
  timestamp: undefined,
  datetime: undefined,
  currency: 'USDT',
  amount: 10,
  fromAccount: 'swap',
  toAccount: 'spot',
  status: 'Success'
}
```

### CODE REVIEW

File: js/coinex.js [hunk 1]
```diff
@@ -3092,6 +3093,49 @@ module.exports = class coinex extends Exchange {
...
+    parseTransfer (transfer, currency = undefined) {
+        return {
+            'id': undefined,
+            'timestamp': undefined,
+            'datetime': undefined,
+            'currency': currency,
+            'amount': undefined,
+            'fromAccount': undefined,
+            'toAccount': undefined,
+            'status': this.safeString (transfer, 'message'),
```
Comment (kroitor) [inline js/coinex.js:3135]: Please, add parseTransferStatus here (`ok / failed`).

File: js/coinex.js [hunk 2]
```diff
@@ -3125,7 +3163,7 @@ module.exports = class coinex extends Exchange {
             'amount': this.safeNumber (transfer, 'amount'),
             'fromAccount': fromAccount,
             'toAccount': toAccount,
-            'status': this.safeString (transfer, 'message'),
+            'status': this.parseTransferStatus (transfer, 'code'),
```
[the remark this answers is not in the document]
Comment (Dan-krm) [reply js/coinex.js:3166]: Okay I see, transfer returns the code as an integer, do you still want me to use safeString or just use safeInteger?

[the remark this answers is not in the document]
Comment (kroitor) [reply js/coinex.js]: I've changed it to strings, jfyi.

[the remark this answers is not in the document]
Comment (Dan-krm) [reply js/coinex.js]: Okay sounds good, looking at the different exchanges implementations of parseTransferStatus, it varies slightly so I wasn't sure what approach would be best

**Pull request merged**